cis_v600/section_2.pp

```diff
@@ -0,0 +1,331 @@
+  query         = query.account_alternate_contact_security_registered
+  documentation = file("./cis_v600/docs/cis_v600_2_2.md")
+
+  tags = merge(local.cis_v600_2_common_tags, {
+    cis_item_id = "2.2"
+    cis_level   = "1"
+    cis_type    = "manual"
+    service     = "AWS/IAM"
+  })
+}
+
+control "cis_v600_2_3" {
+  title         = "2.3 Ensure no 'root' user account access key exists"
+  description   = "The 'root' user account is the most privileged user in an AWS account. AWS Access Keys provide programmatic access to a given AWS account. It is recommended that all access keys associated with the 'root' user account be deleted."
+  query         = query.iam_root_user_no_access_keys
+  documentation = file("./cis_v600/docs/cis_v600_2_3.md")
+
+  tags = merge(local.cis_v600_2_common_tags, {
+    cis_item_id = "2.3"
+    cis_level   = "1"
+    cis_type    = "automated"
+    service     = "AWS/IAM"
+  })
+}
+
+control "cis_v600_2_4" {
+  title         = "2.4 Ensure MFA is enabled for the 'root' user account"
+  description   = "The 'root' user account is the most privileged user in an AWS account. Multi-factor Authentication (MFA) adds an extra layer of protection on top of a username and password. With MFA enabled, when a user signs in to an AWS website, they will be prompted for their username and password as well as for an authentication code from their AWS MFA device."
+  query         = query.iam_root_user_account_console_access_mfa_enabled
+  documentation = file("./cis_v600/docs/cis_v600_2_4.md")
+
+  tags = merge(local.cis_v600_2_common_tags, {
+    cis_item_id = "2.4"
+    cis_level   = "1"
+    cis_type    = "automated"
+    service     = "AWS/IAM"
+  })
+}
+
+control "cis_v600_2_5" {
+  title         = "2.5 Ensure hardware MFA is enabled for the 'root' user account"
+  description   = "The 'root' user account is the most privileged user in an AWS account. MFA adds an extra layer of protection on top of a username and password. With MFA enabled, when a user signs in to an AWS website, they will be prompted for their username and password as well as for an authentication code from their AWS MFA device. For Level 2, it is recommended that the 'root' user account be protected with a hardware MFA."
+  query         = query.iam_root_user_hardware_mfa_enabled
+  documentation = file("./cis_v600/docs/cis_v600_2_5.md")
+
+  tags = merge(local.cis_v600_2_common_tags, {
+    cis_item_id = "2.5"
+    cis_level   = "2"
+    cis_type    = "manual"
+    service     = "AWS/IAM"
+  })
+}
+
+control "cis_v600_2_6" {
+  title         = "2.6 Eliminate use of the 'root' user for administrative and daily tasks"
+  description   = "With the creation of an AWS account, a 'root user' is created that cannot be disabled ordeleted. That user has unrestricted access to and control over all resources in the AWS account. It is highly recommended that the use of this account be avoided for everyday tasks."
+  query         = query.iam_root_last_used
```

Comment:
@khushboo9024 could you please check if `iam_root_last_used` is the correct query to be used here?